src/server/controller/file/index.ts

```diff
@@ -0,0 +1,10 @@
+import { AssessmentFileRepository } from '@server/repository/assessment/file'
+
+import { createAssessmentFile } from './createAssessmentFile'
+
+export const FileController = {
+  createAssessmentFile,
+  getAssessmentFile: AssessmentFileRepository.getOne,
+  getAssessmentFiles: AssessmentFileRepository.getMany,
+  removeAssessmentFile: AssessmentFileRepository.remove,
```

Comment:
If we add the activity log to create file , then also to remove file should be added for consistency

Reply:
done